draft/webxdc-dev-reference.md

```diff
@@ -162,7 +162,18 @@ The following example shows an input field and  every input is show on all peers
 </html>
 ```
 
-For a more advanved example, see https://github.com/r10s/webxdc-poll/ .
+[Webxdc Development Tool](https://github.com/deltachat/webxdc-dev)
+offers an **Webxdc Simulator** that can be used in many browsers without any installation needed.
+You can also use that repository as a template for your own app -
+just clone and start adapting things to your need.
+
+
+### Advanced Examples
```

Comment:
maybe include a link to webxdc topic in github to discover webxdc projects:

https://github.com/topics/webxdc

Reply:
nice! great idea! was always wondering if these tags can be used for sth. useful :)

Reply:
i created a pr at https://github.com/deltachat/deltachat-core-rust/pull/3021 - but please also just do a pr directly there if you think, some of the other apps should be added as a direct link :)